packages/babel-plugin-transform-remove-console/src/index.js

```diff
@@ -47,22 +50,40 @@ module.exports = function({ types: t }) {
     );
   }
 
-  function isConsole(memberExpr) {
+  function isExcluded(property, state) {
```

Comment:
don't pass in the whole `state`. Just pass in whatever is required by this function. This function shouldn't worry about the options being available at `state.opts.exclude`. Just pass in the exclude array.